src/main/resources/standardTestCases/empty_string/empty_string.wdl

```diff
@@ -0,0 +1,33 @@
+task hello {
+  command {
+    echo "Hello!"
+  }
+  output {
+    String empty = ""
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+task goodbye {
+  String emptyInputString
+  command {
+    echo "${emptyInputString}"
+  }
+  output {
+    String empty = read_string(stdout())
+  }
+  runtime {
+   docker: "ubuntu:latest"
+  }
+}
+
+workflow hello {
+  call hello
```

Comment:
Theoretically it's testing both an empty string literal and an interpolated empty string. Of course we also have separate testing for string interpolation. If only we got extra credit on test coverage for the depth of testing as well. 

I'm going to leave this as is, at least for now